subsys/net/lib/wifi_mgmt_ext/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+zephyr_library_sources(wifi_mgmt_ext.c)
+
+zephyr_library_include_directories(${NCS_DIR}/modules/lib/hostap/wpa_supplicant/)
+zephyr_library_include_directories(${NCS_DIR}/modules/lib/hostap/src/)
+zephyr_library_include_directories(${NCS_DIR}/modules/lib/hostap/src/utils)
```

Comment:
Why are those includes here, `hostap` already provides include dirs to Zephyr globally:
https://github.com/nrfconnect/sdk-hostap/blob/main/zephyr/CMakeLists.txt#L22

Reply:
Right, I'll remove the first two. The utils folder is not exposed by default.

Reply:
Well, I'm not sure what't the rationale in `wpa_supplicant` behind exposing certain direcories (i. e. what's supposed to be public and what not), but if you have `hostap/src/` in the include path already, you could simply include whatever is inside `utils` directory by simply prefing the header (i. e. include <utils/xxx.h>). No need to extend the include list.

Reply:
done